gatsby/helpers/pages-graphql-request/pages-graphql-request.ts

```diff
@@ -0,0 +1,26 @@
+const pagesGraphQLRequest = (): string => `
+query MyQuery {
+  allMdx(
```

Comment:
Let's use same style as in bsa website and also rename pages to lectures. Don't forget to rename file
```suggestion
const lecturesGraphQLRequest = (): string => `
{
  allLectures(
```

Reply:
"allMdx" is a command, which tells  graphql to get all .mdx extensions, which are visible due to "gatsby-source-filesystem" module in gatsby-config.ts.
However, renamig approved